tools/python/boutiques/searcher.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, query, verbose=False, sandbox=False, max_results=None,
         if max_results is None:
             self.max_results = 10
 
+        # Zenodo will error if asked for more than 9999 results
+        if(int(self.max_results) > 9999):
+            self.max_results = "9999"
```

Comment:
Shouldn't this be a number?

Reply:
Yes, currently it's not but I'll update it.